### PR TITLE
JavaScript Capstone

### PR DESCRIPTION
**Pull request by nicupop729**
### For these project, the following features were implemented:

We had build these interfaces:
 - The home page.
 - The comments popup.
 - The reservations popup.

**Home page:**
-   When the page loads, the webapp retrieves data from:
-   The forkify API and shows the list of pizza on screen.
-   The Involvement API to show the item likes.
-   When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
-   When the user clicks on the "Comments" button, the Comments popup appears.
-   When the user clicks on the "Reservations" button, the Reservations popup appears.
-   Home page header and navigation similar to the given mockup.
-   Home page footer similar to the given mockup.

  **Comments popup:**
- When the popup loads, the webapp retrieves data from:
-   The forkify API and shows details about the selected item.
-   The Involvement API to show the item comments.
-   When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.

  **Reservations popup:**
  

- When the popup loads, the webapp retrieves data from:
-   The forkify API and shows details about the selected item.
-   The Involvement API to show the item reservations.
-   When the user clicks on the "Reserve" button, the data is recorded in the Involvement API and the screen is updated.

**Counters:**

We have counters in all the interfaces that show:
 - The number of items (home).
 - The number of comments (comments popup).
 - The number of reservations (reservations popup).
 - These count functions are covered with unit tests using Jest.

**Technical set up:**

 - Set up the repository on GitHub and use Gitflow.
 - Set up webpack.
 - Set up a JavaScript testing library (Jest).

**[Presentation video](https://drive.google.com/file/d/1RO82LzCzskc9ShmVl8kqquXy3U56YHPc/view?usp=sharing)**

### CODE REVIEW

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi team, 

Your video presentation is quite good and friendly I really enjoyed it. 👏👌✔️
The README is descriptive. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [x] Clicking the `like` icon invokes the render function for the whole page. Kindly make sure it only affects the clicked item (to only the update the like counter).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- _N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/display-comments.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    e.preventDefault();
+    sendNewComm(pizzaId, inputName.value, inputMess.value);
+    inputName.value = '';
+    inputMess.value = '';
+    setTimeout(() => {
+      commentsList.innerHTML = '';
+      getCommArray(pizzaId, commentsList);
+      commCounterDisplay(commentCounterApi(pizzaId), comments);
+    }, 500);
+  });
```
Comment (henatan99) [inline src/display-comments.js]: - [x] OPTIONAL: Everything looks great, but it seems that when a user submits a comment, the comment shows up only after a refresh. Kindly work on this in your spare time. :pray:

**Review by henatan99 — APPROVED** [review]
Hi @nicupop729 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/display-comments.js (hunk 1 above)
Comment (nicupop729) [reply src/display-comments.js]: **Thanks a lot for the feedback.**
It seems that half a second does not work all the time, so I change it to 1 sec. So now never miss a new comment. And of course, without refresh.

**Pull request merged**